projects/chaos/src/chaos_can.c

```diff
@@ -0,0 +1,60 @@
+#include "chaos_can.h"
+
+#include <stddef.h>
+
+#include "can.h"
+#include "can_ack.h"
+#include "can_msg_defs.h"
+#include "can_unpack.h"
+#include "chaos_events.h"
+#include "event_queue.h"
+#include "misc.h"
+#include "status.h"
+
+#define NO_DATA 0
+#define CHAOS_CAN_NUM_RX_HANDLERS 1
+
+static CANStorage s_can_storage;
+static CANRxHandler s_can_rx_handlers[CHAOS_CAN_NUM_RX_HANDLERS];
+
+// TODO(ELEC-105): Determine if there are any other messages we care about. BPS_FAULT comes to mind
```

Comment:
Per regs BPS fault should kill power immediately. The driver can just coast to safety and stop using mechanical braking + steering.

Reply:
OK I've stubbed this out for now and added a test I'll add this to the Sequencer/GPIO module in #131 once this PR gets merged since it adds the event to the event list.